SoObjects/SOGo/SOGoContentObject.m

```diff
@@ -246,14 +246,12 @@ - (NSException *) saveComponent: (id) theComponent
 
 - (NSException *) copyToFolder: (SOGoGCSFolder *) newFolder
 {
-  [self subclassResponsibility: _cmd];
-
-  return nil;
+  return [self subclassResponsibility: _cmd];
 }
 
 - (NSException *) moveToFolder: (SOGoGCSFolder *) newFolder
 {
-  [self subclassResponsibility: _cmd];
+  return [self subclassResponsibility: _cmd];
```

Comment:
We should add here just a `return nil` to make compiler happy because that return command is never called as explained above.